mk/docker.mk

```diff
@@ -106,7 +106,7 @@ docker/save: $(patsubst %,docker/%/save,$(ALL_RELEASE_WITH_ARCH) $(ALL_TEST_WITH
 .PHONY: docker/load
 docker/load: $(patsubst %,docker/%/load,$(ALL_RELEASE_WITH_ARCH) $(ALL_TEST_WITH_ARCH))
 .PHONY: docker/tag
-docker/tag: docker/tag/test docker/tag/release
+docker/tag: docker/tag/test docker/tag/release ## Tag local arch containers with the version with the arch (this is mostly to use non multi-arch images as if they were released images in e2e tests)
```

Comment:
I'm not sure I follow, why do we want to use non multi-arch images?

Reply:
Because with non multi-arch images you only need to build 50% of the things (only the host arch) so things are 2x faster :)